### PR TITLE
Master Branch Changelogs Only

### PR DESCRIPTION
**Pull request by RemieRichards**
@MrStonedOne 

because april fools meme prs having their changelogs in the game is very weird. (like https://github.com/tgstation/-tg-station/pull/16324 and https://github.com/tgstation/-tg-station/commit/6520a6ee0977eaa7c4a35de17b3c5e38c188539d for example)

### CODE REVIEW

**Comment by RemieRichards** [comment]
Yay a typo

File: tools/github_webhook_processor.php [hunk 1]
```diff
@@ -247,7 +252,7 @@ function checkchangelog($payload, $merge = false) {
 		}
 	}
 	
-	if (!count($changelogbody))
+	if (!count($changelogbody) || $target_branch_ref != "master")
```
Comment (Iamgoofball) [inline tools/github_webhook_processor.php]: un-hardcode this and switch it for a config option

Comment (RemieRichards) [inline tools/github_webhook_processor.php]: a config option where? the /game/'s config?

Comment (Giacom) [inline tools/github_webhook_processor.php]: You could just make it a constant variable like $apiKey

Comment (RemieRichards) [inline tools/github_webhook_processor.php]: I suppose, but in this instance it's still only changing one variable's contents (well, you're changing just a literal here but butts I'm just making a point) but I suppose that's easier to locate given it's at the top of the file, so I'll go ahead and do it.

**Comment by Supermichael777** [comment]
they do. i know because my bee gun changelog confused a few people

**Comment by RemieRichards** [comment]
Aha! so it did, thanks @Supermichael777

**Comment by MrStonedOne** [comment]
Well then, I'll take a look, but the script SHOULD have sent the changelog to the same branch as the target branch, that's the ideal solution and I was sure I tested and confirmed it was doing that.

**Comment by RemieRichards** [comment]
@MrStonedOne's fixing a different issue that fixes this one too, so, here:
https://github.com/tgstation/-tg-station/pull/16454

**Pull request closed without merging**